assets/dev/scss/frontend/_container.scss

```diff
@@ -30,7 +30,8 @@
 	--position: relative;
 	--z-index: revert;
 	--overflow: visible;
-	--gap: var( --widgets-spacing, 20px ); // Use the gap from `Site Settings` or default to 20px.
+	--column-gap: var( --widgets-spacing, 20px ); // Use the gap from `Site Settings` or default to 20px.
+	--row-gap: var( --widgets-spacing, 20px ); // Use the gap from `Site Settings` or default to 20px.
```

Comment:
The combination of having two gap values inside the --row seemed to cause problems. Therefore I split up the values for the column and the row gap.

Reply:
Shouldn't removing these properties in  `assets/dev/scss/frontend/_global.scss` be better?
The problem is here.
```
		column-gap: var( --grid-column-gap, var( --gap ) );
		row-gap: var( --grid-row-gap, var( --gap ) );
```

Reply:
Fixed.